### DISCUSSION TITLE
Batteries Version Display - B1, B2, B3

### DISCUSSION THREAD

**Discussion by zh17home** [WillCodeForCats/solaredge-modbus-multi#600] [General]
Hello,

From my view in HA:
B1 - SMCU 1.1.35 DCDC 0.1.85 BMS 0.6 - Almost OK -> should be BMS 0.6.9 (missing .9)
B2 - SMCU 1.1.35 DCDC 0.0.81 BMS 0.6 - NOK  for DCDC / BMS
B3 - SMCU 1.1.29 DCDC 0.0.81 BMS 0.6 - NOK for DCDC / BMS

From Monitoring:
B1 - SMCU 1.1.35 DCDC 0.1.85 BMS 0.6.9
B2 - SMCU 1.1.35 DCDC 0.1.85 BMS 0.6.9
B3 - SMCU 1.1.29 DCDC 0.1.85 BMS 0.6.9 (Is missing the upgrade for SMUC, I will try later).

Thanks

**Comment by WillCodeForCats** [comment]
The battery version string is limited to 32 characters over modbus.

**Reply by zh17home** [reply]
okok Thanks Willcode

**Comment by zh17home** [comment]
Now shows  BMS 0.7 :)

<img width="1372" alt="image" src="https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/65257232/302e7ae8-51a4-45b7-9708-7a1a424723e3">

From Monitor, still 0.6.9
<img width="413" alt="image" src="https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/65257232/b0cbf877-f9a7-406f-adc5-62ea7fb3e646">
